modules/Microsoft.Storage/storageAccounts/.test/common/deploy.test.bicep

```
@@ -0,0 +1,201 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'ms.storage.storageaccounts-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'stscom'
```

```suggestion
param serviceShort string = 'ssacom'
```

... Just to follow the approach we used in all other modules (i.e. capital/first letters of the providernamespace+resourceType+foldername)